MarsOnboardingTask/StepDefinitions/LanguagesStepDefinitions.cs

```diff
@@ -0,0 +1,112 @@
+using MarsOnboardingTask.Pages;
+using MarsOnboardingTask.Utils;
+using NUnit.Framework;
+
+
+namespace MarsOnboardingTask.StepDefinitions
+{
+    [Binding]
+    public class LanguagesStepDefinitions : Hooks
```

Comment:
I would extend to Driver class rather then Hooks

Reply:
updated as per the review comments